runner/builder/builder.go

```diff
@@ -45,20 +47,22 @@ func (b *Builder) Create(req *api.CreateRunnerRequest) (*runnerpb.Runner, error)
 	}
 
 	instanceEnv := xos.EnvMergeSlices(srv.Configuration.Env, req.Env)
-	envHash := hash.Dump(hashserializer.StringSlice(instanceEnv))
+	envHash := hash.Sum([]byte(strings.Join(instanceEnv, ",")))
```

Comment:
why not doing the same here too ;)
this way we only have one hashing function, same that events that are not needed as a cosmos address but not bad representation anyway